samples/javascript_typescript/02.echobot-with-counter/src/bot.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// bot.js is your bot's main entry point to handle incoming activities.
```

Comment:
Do we need this comment? I would prefer a docstring for the class itself, above line 11.